File: tort-core/src/test/java/com/mzr/tort/testsample/domain/dto/FormDto.java
package com.mzr.tort.testsample.domain.dto;

import java.util.HashSet;
import java.util.Set;
import javax.persistence.CascadeType;
import javax.persistence.FetchType;
import javax.persistence.ManyToOne;
import javax.persistence.OneToMany;

import com.mzr.tort.core.domain.LongIdEntity;

public class FormDto extends LongIdEntity {

    private String name;

    private UniversityDto university;

    private Set<StudentDto> students = new HashSet<>();

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public UniversityDto getUniversity() {
        return university;
    }

    public void setUniversity(UniversityDto university) {
        this.university = university;
    }

    public Set<StudentDto> getStudents() {
        return students;
    }

    public void setStudents(Set<StudentDto> students) {
        this.students = students;
    }
}

File: sample-tort-app/src/main/kotlin/com/mzr/tort/sample/domain/dto/FormDto.kt
package com.mzr.tort.sample.domain.dto

import com.mzr.tort.core.domain.LongIdEntity
import com.mzr.tort.sample.domain.entity.Student
import com.mzr.tort.sample.domain.entity.University
import java.util.*

class FormDto : LongIdEntity() {

    var name: String? = null

    var students: Set<StudentDto> = HashSet()
}
